File: src/main/java/com/trade/securities/application/port/in/GetFscNewShareCertificateUseCase.java
package com.trade.securities.application.port.in;

import java.util.Map;

public interface GetFscNewShareCertificateUseCase {
    Map<String, Object> getNewShareCertificates(int pageNo, int numOfRows, String basDt, String itmsNm,
            String likeItmsNm);
}

File: src/main/kotlin/com/trade/securities/application/port/in/GetFscNewShareCertificateUseCase.kt
package com.trade.securities.application.port.`in`

interface GetFscNewShareCertificateUseCase {
    fun getNewShareCertificates(pageNo: Int, numOfRows: Int, basDt: String, itmsNm: String?, likeItmsNm: String?): Map<String, Any?>
}
